glusterd2/xlator/validation.go

```diff
@@ -6,6 +6,7 @@ import (
 	"strings"
 
 	"github.com/gluster/glusterd2/glusterd2/volume"
+	quota "github.com/gluster/glusterd2/plugins/quota"
```

Comment:
nit: You don't need the alias here. You can use the import directly.

Reply:
done